packages/mobile/src/functions/handleFormSubmit.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import {ToastAndroid} from 'react-native';
+import postUser from '../api/PostUser';
+
+const handleFormSubmit = async (values: any, actions: any) => {
+  const user = {
+    email: values.email,
+    password: values.password,
+  };
+  actions.resetForm();
+  try {
+    const res = await postUser(user);
+    console.log(res);
+    ToastAndroid.showWithGravity(
+      'Rejestracja pomyśla',
+      ToastAndroid.LONG,
+      ToastAndroid.CENTER,
+    );
+  } catch (err) {
+    console.log(err.response.data.statusCode);
+    if (err.response.data.statusCode === 409) {
```

Comment:
statusCode 409 to za mało by stwierdzić że to użytkownik się powtarza bo 409 to Conflict. Nie masz tam kodu błędu w body lub message po którym mógłbyś jednoznacznie stwierdzić że to użytkownik się powtórzył?

Reply:
A co się stanie jak serwer się wywali  i dostaniesz 500?